### DISCUSSION TITLE
How to detect  a if a not logged mac conect to my internal network?

### DISCUSSION THREAD

**Discussion by sergaar** [Security-Onion-Solutions/securityonion#2043] [Unsupported Versions] [answered]
I was looking for a way to detect if a not logged mac conect to my internal network, I saw that with zeek y can see the computers's MACs that connected to my network but I didn`t found a way to log known macs, and generate alert if a not logged mac conected to my network.
I designed a script which analyce conn.log and it warn if a unknown mac connect, this script needs a file with the known macs, and it´s very tedious to write mac by mac,

Is there a way to do this without a script? 

Thanks

**Comment by weslambert** [comment, marked as answer]
Off the top of my head, you could use Zeek logging, in conjunction with Zeek scripting, and generating notices, or with an Elastalert rule..
